<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>691</width>
     <height>601</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>691</width>
     <height>601</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="loginSubmit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>318</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="passwordLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="emailLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailLineEdit">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>370</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unregisterButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Unregister</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="welcomeLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>47</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Items</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Shopping List</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>671</width>
       <height>491</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>652</width>
        <height>489</height>
       </rect>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>326</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>0</y>
         <width>321</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="homeLogout">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeClearButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Item:</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>216</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="homeAddItemLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="homeAddButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="regBackButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>360</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="regRegisterButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>360</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="regUsernameLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="regUsernameLineEdit">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="regEmailLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="regEmailLineEdit">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="regPasswordLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="regPasswordLineEdit">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="regPhoneLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone#:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="regPhoneLineEdit">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QSplitter" name="splitter_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="unregisterEmailLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="unregisterEmailLineEdit">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="unregisterBackButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unregisterUnregisterButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>300</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Unregister</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="unregisterPasswordLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="unregisterPasswordLineEdit">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>228</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>loginSubmit</tabstop>
  <tabstop>unregisterButton</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>regBackButton</tabstop>
  <tabstop>regRegisterButton</tabstop>
  <tabstop>regEmailLineEdit</tabstop>
  <tabstop>regUsernameLineEdit</tabstop>
  <tabstop>regPasswordLineEdit</tabstop>
  <tabstop>regPhoneLineEdit</tabstop>
  <tabstop>unregisterBackButton</tabstop>
  <tabstop>unregisterUnregisterButton</tabstop>
  <tabstop>unregisterEmailLineEdit</tabstop>
  <tabstop>unregisterPasswordLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
